cratedb_toolkit/io/iceberg.py

```
@@ -0,0 +1,121 @@
+
+    # Note: This was much slower.
+    # table.to_polars().collect(streaming=True).write_database(table_name=table_address.fullname, connection=engine, if_table_exists="replace")
+
+
+def to_iceberg(source_url, target_url, progress: bool = False):
+    """
+    Synopsis
+    --------
+    export CRATEDB_CLUSTER_URL=crate://crate@localhost:4200/testdrive/demo
+    ctk load table "iceberg://./var/lib/iceberg/?catalog=default&table=taxi_dataset"
+    ctk save table "file+iceberg:var/lib/iceberg/default.db/taxi_dataset/metadata/00001-dc8e5ed2-dc29-4e39-b2e4-019e466af4c3.metadata.json"
```

Ruff (0.11.9)

Line too long (139 > 120)

(E501)